ocs_ci/utility/utils.py

```diff
@@ -934,7 +934,120 @@ def parse_html_for_email(soup):
     main_header.string.replace_with('OCS-CI RESULTS')
 
 
-def email_reports():
+def add_squad_analysis_to_email(session, soup):
+    """
+    Add squad analysis to the html test results used in email reporting
+
+    Args:
+        session (obj): Pytest session object
+        soup (obj): BeautifulSoup object
```

Comment:
BeautifulSoap object of what?  Maybe worth to add like of HTML report data?

Reply:
Copy&paste from other method, fixed in 21b7e01.